gitops/snapshot.go

```diff
@@ -77,6 +77,9 @@ const (
 	// SnapshotGitSourceRepoURLAnnotation contains URL of the git source repository (usually needed for forks)
 	SnapshotGitSourceRepoURLAnnotation = "test.appstudio.openshift.io/source-repo-url"
 
+	// SnapshotGitSourceRepoBranchAnnotation contains branch of the git source repository (usually needed for forks)
+	SnapshotGitSourceRepoBranchAnnotation = "test.appstudio.openshift.io/source-repo-branch"
```

Comment:
Do we have `/source-repo-branch` exported by PAC? I know we have `/source-branch` exported by PAC

Reply:
yes, 
double check the[ PaC release v0.27.1](https://github.com/openshift-pipelines/pipelines-as-code/commit/789342d31e705c1c179442968ee8b3b38af7776f#diff-5c2a268808defe023737ef3597791fad8ede63bb8adc22721ae25660ad7a361fR35) , it's  `/source-branch` , 
fixed